docs/migration-guides/_toc.json

```diff
@@ -9,6 +9,11 @@
       {
         "title": "Migrate to Qiskit 2.0",
```

Comment:
Making sure it's intentional to have the older Migrate from Backend V1 to V2 as a child of the Migrate to Qiskit 2.0 parent?

Reply:
@ElePT can confirm, but I think that was her intention, or at least to reference it from within the 2.0 migration guide.